algorithms-modules/algorithms-miscellaneous-9/src/test/java/com/baeldung/algorithms/permutation/StringPermutationUnitTest.java

```diff
@@ -10,6 +8,8 @@
 import static org.junit.jupiter.api.Assertions.assertFalse;
 import static org.junit.jupiter.api.Assertions.assertTrue;
 
+import org.junit.jupiter.api.Test;
+
```

Comment:
Do we need this?

Reply:
> this is a minor thing we can do at any time

Including never. It just adds noise to this PR. However, leave it be.

I appreciate the "boy scout rule" here, cleaning up the code. Perhaps this isn't the project to do that in. But it's not worth taking time to revert it.